### ISSUE TITLE
Data returned from rendering method is missing textures

### ISSUE THREAD

**Issue by zhuyifengzju** [owl-project/NVISII#94]
I visualized data returned from `render` method, and it seems that the texture of the table is not rendered properly.

![Screenshot from 2021-02-18 21-59-59](https://user-images.githubusercontent.com/21077484/108456005-ede4e900-7234-11eb-8697-7aa1e1eb6c19.png)
![image](https://user-images.githubusercontent.com/21077484/108456017-f5a48d80-7234-11eb-85a3-bbb71fa555ea.png)

**Comment by n8vm** [comment]
Hey @zhuyifengzju,

Is this after upgrading from a previous version? 

Would you be able to share your script and data with me? Possibly over a Google drive link or similar? That would help me repro locally, and then from there I could solve the issue faster.

**Comment by zhuyifengzju** [comment]
Yes, this still happens after I upgrade to nvisii from visii. I'll try to write a simple case of the table and share with you.

**Comment by n8vm** [comment]
@zhuyifengzju that dataset appears to be a Franka URDF. Guessing the table texture is being pulled in from an OBJ material? 

Does this issue only occur then using the “render” function? Do you see the same issue when rendering in interactive mode? ( Using nvisii.initialize(headless = False) )

**Comment by zhuyifengzju** [comment]
Yes, either headless or not having the same problem. Yes the table texture is from a obj material. The visualization and image saving is fine though. For you reference, I am using `import_scene` function to import the mesh.

**Comment by n8vm** [comment]
I believe I'm able to reproduce a similar issue to the one you have above. Seems like if a user issues many commands in a row followed by a call to render, a race condition can occur where some of the previous python scene edits are lost. I think I have a fix now, where I first process the entire command queue before starting the render. Going to try getting that into the next release along with another bug fix.

**Comment by zhuyifengzju** [comment]
Thanks a lot!

**Comment by n8vm** [comment]
@zhuyifengzju Appologies for the delay. Would you be willing to test the new version of nvisii on pip (version 1.1) and let me know if your issue here is solved?
